akka-docs/src/main/paradox/typed/replicated-eventsourcing.md

```diff
@@ -314,7 +314,7 @@ Scala
 Java
 :  @@snip [ReplicatedShardingTest.java](/akka-cluster-sharding-typed/src/test/java/jdocs/akka/cluster/sharding/typed/ReplicatedShardingCompileOnlySpec.java) { #bootstrap }
 
-This will run a single instance of sharding and the replicas will be differentiated by having the replica id in the sharding entity id.
+This will run an instance of sharding and per replica and each entity id contains the replica id and the type name.
 Replicas could be on the same node if they end up in the same shard or if the shards get allocated to the same node.
```

Comment:
I think some more adjustments are needed in the Direct Replication section

Reply:
I got confused and updated them in the other PR: https://github.com/akka/akka/pull/29492